### PR TITLE
feat: mprocs setup for guardian-ui dev

### PR DESCRIPTION
**Pull request by EthnTuttle**
`just run-ui` brings up mprocs with a 2 instance federation via `run-ui.sh` and 2 instances of the Guardian UI

### CODE REVIEW

**Review by elsirion — APPROVED** [review]
Tested, works for me, really cool!

It could use some docs/help in case a user is "holding it wrong" though:

* Needs to be run in `nix develop .#fedimint-ui` shell (can we warn if that's not the case, e.g. `yarn` isn't available)
* `yarn install` needs to be run before (is there a way to check and warn the user if that hasn't been done?)

But that could be a follow-up PR :)

File: misc/mprocs-ui.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+procs:
+  run-ui-federation:
+    shell: ./scripts/run-ui.sh new
+  coordinator:
+    cwd: fedimint-ui/apps/guardian-ui/
+    shell: yarn dev
```
Comment (justinmoon) [inline misc/mprocs-ui.yaml:6]: need yarn install

**Comment by EthnTuttle** [comment]
Do 4 guardians minimum.
Or pass in # of guardians.

**Comment by elsirion** [comment]
I think just having a check in `./scripts/run-ui.sh` and warning the user/pointing into the right direction in the first mprocs window would help a lot

**Pull request merged**